src/stan-users-guide/proportionality-constants.Rmd

```diff
@@ -0,0 +1,150 @@
+# Proportionality Constants  {#proportionality-constants.chapter}
+
+When evaluating a likelihood or prior as part of the log density computation in
+MCMC, variational inference, or optimization, it is usually only necessary to
+compute the functions up to a proportionality constant (or similarly compute
+log densities up to an additive constant). In MCMC this comes from the fact that
+the distribution being sampled does not need to be normalized (similarly
+the distribution does not need normalized to perform variational inference or
+do optimizations).
+
+There are three different syntaxes to work with distributions in Stan. The way
+to select between them is by determining if the proportionality constants are
+necessary.
+
+The first two syntaxes drop proportionality constants:
+
+```
+x ~ normal(0, 1);
+target += normal_lupdf(x | 0, 1);
```

Comment:
The first time we show this syntax should we say that "u" stands for "unnormalized" to make the connection between the name and the purpose clear?